test/system/550-pause-process.bats

```diff
@@ -6,6 +6,14 @@
 load helpers
 load helpers.sig-proxy
 
+function setup_file() {
```

Comment:
sorry that happens when I don't test my diff's, looks like it needs a `load helpers.registry` at the top to import the function